service/rpc/servicer.go

```diff
@@ -419,6 +419,320 @@ func (s *Servicer) BlockTransaction(ctx context.Context, request *types.BlockTra
 	}, nil
 }
 
+func (s *Servicer) Call(
+	ctx context.Context,
+	callRequest *types.CallRequest,
+) (*types.CallResponse, *types.Error, error) {
+	switch callRequest.Method {
+	case isAccount:
+		address, ok := callRequest.Parameters["address"].(common.Address)
+		if !ok {
+			err := errors.New("failed to parse 'address' as common.Address")
+			return nil, LogErrValidation(err), err
+		}
+
+		callOpts, err := buildCallOpsFromCallRequest(ctx, callRequest)
+		if err != nil {
+			return nil, LogErrValidation(err), err
+		}
+
+		registry, err := registry.New(s.cc)
+		account, err := registry.GetAccountsContract(ctx, nil)
+		if err != nil {
+			return nil, LogErrValidation(err), err
+		}
+
+		isAccount, err := account.IsAccount(callOpts, address)
+		if err != nil {
+			return nil, LogErrValidation(err), err
+		}
+
+		result := map[string]interface{}{
+			"isAccount": isAccount,
+		}
+
+		return &types.CallResponse{
+			Result:     result,
+			Idempotent: true,
+		}, nil, nil
+
+	case getVoteSigner:
+		address, ok := callRequest.Parameters["address"].(common.Address)
+		if !ok {
+			err := errors.New("failed to parse 'address' as common.Address")
+			return nil, LogErrValidation(err), err
+		}
+
+		callOpts, err := buildCallOpsFromCallRequest(ctx, callRequest)
+		if err != nil {
+			return nil, LogErrValidation(err), err
+		}
+
+		registry, err := registry.New(s.cc)
+		account, err := registry.GetAccountsContract(ctx, nil)
+		if err != nil {
+			return nil, LogErrValidation(err), err
+		}
+
+		voteSigner, err := account.GetVoteSigner(callOpts, address)
+		if err != nil {
+			return nil, LogErrValidation(err), err
+		}
+
+		result := map[string]interface{}{
+			"voteSigner": voteSigner,
+		}
+
+		return &types.CallResponse{
+			Result:     result,
+			Idempotent: true,
+		}, nil, nil
+
+	case canReceiveVotes:
+		group, ok := callRequest.Parameters["group"].(common.Address)
```

Comment:
Would it make sense to add some validation to check whether this is an address registered with Elections.sol?

How does the contract function respond if this address is not registered @yorhodes? (https://github.com/celo-org/celo-monorepo/blob/master/packages/protocol/contracts/governance/Election.sol#L857)